### PR TITLE
{lib}[foss/2018b] TensorFlow v1.11.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires https://github.com/easybuilders/easybuild-easyblocks/pull/1553 (PythonBundle easyblock) + https://github.com/easybuilders/easybuild-easyblocks/pull/1559 (fixes for TensorFlow easyblock)

### CODE REVIEW

**Comment by akesandgren** [comment]
close/open to kick travis

**Comment by boegel** [comment]
@akesandgren No point in kicking Travis until https://github.com/easybuilders/easybuild-easyblocks/pull/1553 is merged too...

**Comment by akesandgren** [comment]
Doh, I didnt check properly...

File: easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.11.0-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        'checksums': ['ef2e482c4336fcf7180244d06f4374939099daa3183816e82aee7755af35b754'],
+        'modulename': 'keras_preprocessing',
+    }),
+    (name, version, {
+        'source_urls': ['https://github.com/tensorflow/tensorflow/archive/'],
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'patches': [
+            'TensorFlow-1.11.0_swig-env.patch',
+            'TensorFlow-1.11.0_remove-msse-hardcoding.patch',
+            'TensorFlow-1.11.0_fix-ToString.patch',
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.11.0-foss-2018b-Python-3.6.6.eb]: are these patches to 1.11.0 missing from this PR or was the idea to reuse the patches for 1.10.0?

**Comment by boegel** [comment]
@migueldiascosta Woops, they were missing, indeed... Can't reuse the ones we have for TF 1.10.x, since they don't apply anymore (of course they don't, it's TF...). Just pushed them.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2007.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/07b7821fc3b0d308a7e1a1a6fd932504 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2430.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d1d302d78dd75ea7067e231be02a0fb9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3101.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/1335ce6d0c08f0c9fa63fd096cd6b2f8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2698.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/78199e5c5db403a754f1623d397d309a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2037.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b905cee20d6c128376004befc4c56755 for a full test report.

**Comment by boegel** [comment]
@akesandgren Good to go?

**Pull request merged**